### ISSUE TITLE
node_exporter: Download node_exporter step failing due to an extra space

### ISSUE THREAD

**Issue by jmansar** [prometheus-community/ansible#15]
I get the following error when trying to execute a playbook with the latest revision of `node_exporter` role. 

```
fatal: [REDACTED-> localhost]: FAILED! => 
{
  "attempts": 5, 
  "changed": false, 
  "dest": "/tmp/node_exporter-1.5.0.linux-amd64.tar.gz", 
  "elapsed": 0, 
  "msg": "An unknown error occurred: URL can't contain control characters. '/prometheus/node_exporter/releases/download/v1.5.0/ node_exporter-1.5.0.linux-amd64.tar.gz' (found at least ' ')", 
  "url": "https://github.com/prometheus/node_exporter/releases/download/v1.5.0/ node_exporter-1.5.0.linux-amd64.tar.gz"
}
```

It looks like it's due to the space charater in the download url: 
`https://github.com/prometheus/node_exporter/releases/download/v1.5.0/ node_exporter-1.5.0.linux-amd64.tar.gz`.

**Comment by stevo-f3** [comment]
Maybe new line on https://github.com/prometheus-community/ansible/blob/main/roles/node_exporter/tasks/install.yml#L28

It's not there in the original source https://github.com/cloudalchemy/ansible-node-exporter/blob/master/tasks/install.yml#L24

**Comment by gardar** [comment]
This was fixed in #17
